src/libs/decoders/Makefile.am

```diff
@@ -21,6 +20,12 @@ libdecoders_a_SOURCES = \
 	xxhash.c             \
 	xxhash.h
 
+if USE_OPUS
+	libdecoders_a_SOURCES += opus.c
+	AM_CFLAGS += -DUSE_OPUS $(OPUSFILE_CFLAGS)
+	LDADD += $(OPUSFILE_LIBS)
+endif
```

Comment:
I *suspect* problem might be around here (not sure yet).

Reply:
Argh - also confirm on my side. Well this is annoying and embarrassing; I thought I tested this already; sorry @dreamer .  Will sort this out ASAP.

Reply:
I don't find it embarrassing - I'm rather frustrated by autoconf right now. I literally can't say what is wrong.